OpenRA.Game/FileSystem/InstallShieldCABExtractor.cs

```diff
@@ -404,9 +405,44 @@ public string DirectoryName(uint index)
 			return test;
 		}
 
+		public string GetFirstSubDirectory(string package)
```

Comment:
Ah, I see.  That should not be included in this PR.  Replacing the opaque and arbitrary file IDs with filenames is good, but we are not ready to support non-english disks OR RA2 right now.